src/NUnitEngine/nunit.engine/nunit.engine.csproj

```diff
@@ -21,9 +21,11 @@
   <ItemGroup>
     <Compile Include="..\EngineVersion.cs" LinkBase="Properties" />
     <Compile Include="..\nunit-agent\AgentExitCodes.cs" LinkBase="Agents" />
+    <Compile Include="..\nunit.engine.core\Internal\ExceptionHelper.cs" Link="Internal\ExceptionHelper.cs" />
```

Comment:
As above - can this just reference this class directly, rather than the link, to be consistent with the other Util classes?

Reply:
See earlier comment.